Classes/Domain/Handler/DefaultMigrationHandler.php

```diff
@@ -22,4 +29,19 @@ public function down(Migration $migration): void
     {
         $migration->down();
     }
+
+    public function setConsoleOutput(ConsoleOutput $consoleOutput): void
```

Comment:
You should allow either null as argument, like in `MigrationExecutor` constructor or disallow nullable values in the  `MigrationExecutor` constructor